nodebuilder/header/module.go

```diff
@@ -115,6 +116,7 @@ func ConstructModule(tp node.Type, cfg *Config) fx.Option {
 						p2p.WithMaxAwaitingTime(cfg.Client.MaxAwaitingTime),
 						p2p.WithDefaultScore(cfg.Client.DefaultScore),
 						p2p.WithMaxTrackerSize(cfg.Client.MaxPeerTrackerSize),
+						p2p.WithProtocolSuffix[p2p.ClientParameters](string(modp2p.DefaultNetwork)),
```

Comment:
We must use the provided Network. The same goes for Server. Pls also double-check that we pass the correct one for there pubsub topics

Reply:
done